Helm/opensearch-dashboards/templates/deployment.yaml

```diff
@@ -0,0 +1,156 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "opensearch-dashboards.fullname" . }}
+  labels: {{ include "opensearch-dashboards.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
```

Comment:
`values.yaml` uses `replicaCount` for this field.

Reply:
Changes done in the PR.